ansible/files/admin_api_scripts/pg_upgrade_scripts/initiate.sh

```diff
@@ -41,10 +41,14 @@ LOG_FILE="/var/log/pg-upgrade-initiate.log"
 
 POST_UPGRADE_EXTENSION_SCRIPT="/tmp/pg_upgrade/pg_upgrade_extensions.sql"
 POST_UPGRADE_POSTGRES_PERMS_SCRIPT="/tmp/pg_upgrade/pg_upgrade_postgres_perms.sql"
-OLD_PGVERSION=$(run_sql -A -t -c "SHOW server_version;")
+OLD_PGVERSION=$(pg_config --version | sed 's/PostgreSQL \([0-9]*\.[0-9]*\).*/\1/')
+
+# Skip locale settings if both versions are PostgreSQL 17+ or 17-orioledb
+if ! [[ (("$OLD_PGVERSION" =~ ^17.* || "$OLD_PGVERSION" == "17-orioledb") && ("$PGVERSION" =~ ^17.* || "$PGVERSION" == "17-orioledb")) ]]; then
```

Comment:
Unless I'm mistaken, I think OrioleDB patches postgres won't return `PostgreSQL 17-orioledb`. Or is OrioleDB PostgreSQL built with custom `PG_VERSION`?

Reply:
@za-arthur it is built with custom version. But I ended up removing that because right this moment we do not support upgrade of orioledb -> orioledb I'll need to create a new issue for this.